maintainers/backporting.md

```diff
@@ -0,0 +1,8 @@
+
+# Backporting
+
+Backports are handled by the backport action.
+
+Since GitHub Actions can not trigger actions, the backport PR needs to be re-triggered by another actor. This is achieved by closing and reopening the backport PR.
+
+This specifically affects the `installer_test` check, but note that it only runs after the other tests, so it may take a while to appear.
```

Comment:
```suggestion
This specifically affects the [`installer_test`] check.
Note that it only runs after the other tests, so it may take a while to appear.

[`installer_test`]: https://github.com/NixOS/nix/blob/895dfc656a21f6252ddf48df0d1f215effa04ecb/.github/workflows/ci.yml#L70-L91
```

We discussed at length about [using permalinks in the Nixpkgs manual](https://github.com/NixOS/nixpkgs/pull/121015#issuecomment-1072967188).